lib/fluent/plugin/out_s3.rb

```diff
@@ -268,7 +268,7 @@ def write(chunk)
       begin
         @compressor.compress(chunk, tmp)
         tmp.rewind
-        log.debug { "out_s3: write chunk: {key:#{chunk.key},tsuffix:#{tsuffix(chunk)}} to s3://#{@s3_bucket}/#{s3path}" }
+        log.debug "out_s3: write chunk with metadata #{chunk.metadata} to s3://#{@s3_bucket}/#{s3path}"
```

Comment:
@repeatedly I realized that `chunk.metadata` might only exist for 0.14+ -- is there a better thing to print that will give useful info for all versions?

Reply:
Latest version, 1.0.0 or later, is only for v0.14, so considering v0.12 is not needed I think.